ote/src/cljs/ote/views/route/route_list.cljs

```diff
@@ -79,19 +80,21 @@
          (fn [i {::transit/keys [route-id name]
                  ::modification/keys [created modified] :as row}]
            ^{:key (str "route-" i)}
-           [ui/table-row {:key (str "route-" i) :selectable false :display-border false}
+           [ui/table-row {:key (str "route-" i) :selectable false :display-border false :style {:border-bottom (str "1px solid" colors/gray650)}}
             [ui/table-row-column {:style {:width "18%"}}
              [:a {:href "#"
                   :on-click #(do
                                (.preventDefault %)
                                (e! (fp/->ChangePage :edit-route {:id route-id})))} (t-service/localized-text-with-fallback @selected-language name)]]
             [ui/table-row-column {:style {:width "15%"}} (time/format-timestamp-for-ui modified)]
             [ui/table-row-column {:style {:width "15%"}} (time/format-timestamp-for-ui created)]
-            [ui/table-row-column {:style {:width "13%"}}
-             [ui/icon-button {:href "#"
-                              :on-click #(do
-                                           (.preventDefault %)
-                                           (e! (fp/->ChangePage :edit-route {:id route-id})))}
+            [ui/table-row-column {:style {:width "11%" :padding-left "10px"}}
```

Comment:
padding should probably be rem or em

Reply:
Padding is needed because it moves action icons to left. Icons are sized as px, so px is needed here also.